packages/itwin/grouping-mapping-widget/src/components/GroupingMappingContext.tsx

```diff
@@ -62,6 +64,31 @@ export interface GroupingMappingContextProps {
 const authorizationClientGetAccessToken = async () =>
   (await IModelApp.authorizationClient?.getAccessToken()) ?? "";
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+      // staleTime: 20 * 60 * 1000, // 20 minutes
```

Comment:
Remove commented out code.

Reply:
Actually this stays. Accidently pushed it. Fixed

Reply:
I don't understand. Why do we want to keep commented out configuration?

Reply:
This is the cache time. I disabled the cache to debug. I accidently pushed it.

Reply:
In the reply you said `Fixed`. How?

Reply:
I uncommented it.